scripts/common.py

```diff
@@ -640,29 +640,6 @@ def wait_for_port_to_be_open(port_number):
         sys.exit(1)
 
 
-@contextlib.contextmanager
-def managed_elasticsearch_dev_server():
```

Comment:
Any specific reason for moving this to the bottom of the file?

Reply:
Yup, it logically should go with all the other "managed server" functions.